clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -157,3 +157,118 @@ export const CustomMultiSelect = ({
     </FormControl>
   );
 };
+
+export const CustomCreatableSingleSelect = ({
+  label,
+  isSearchable,
+  isClearable,
+  options,
+  ...props
+}: SelectProps & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
+  const selected = { label: field.value, value: field.value };
+
+  return (
+    <FormControl>
+      <SimpleGrid columns={[1, 2]}>
+        <FormLabel htmlFor={props.id || props.name}>{label}</FormLabel>
+        <CreatableSelect
+          options={options}
+          onBlur={(option) => {
+            if (option) {
+              field.onBlur(props.name);
+            }
+          }}
+          onChange={(newValue) => {
+            if (newValue) {
+              field.onChange(props.name)(newValue.value);
+            } else {
+              field.onChange(props.name)("");
+            }
+          }}
+          name={props.name}
+          value={selected}
+          chakraStyles={{
+            dropdownIndicator: (provided) => ({
+              ...provided,
+              background: "white",
+            }),
+            multiValue: (provided) => ({
+              ...provided,
+              background: "primary.400",
+              color: "white",
+            }),
+            multiValueRemove: (provided) => ({
+              ...provided,
+              display: "none",
+              visibility: "hidden",
+            }),
+          }}
+          isClearable
+        />
+      </SimpleGrid>
+      {isInvalid ? <FormErrorMessage>{meta.error}</FormErrorMessage> : null}
+    </FormControl>
+  );
+};
+
+export const CustomCreatableMultiSelect = ({
+  label,
+  isSearchable,
+  isClearable,
+  options,
+  ...props
+}: SelectProps & FieldHookConfig<string[]>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
+  const selected = field.value.map((v) => ({ label: v, value: v }));
+  const { setFieldValue } = useFormikContext();
+
+  return (
+    <FormControl>
+      <SimpleGrid columns={[1, 2]}>
+        <FormLabel htmlFor={props.id || props.name}>{label}</FormLabel>
+        <CreatableSelect
+          name={props.name}
+          chakraStyles={{
+            dropdownIndicator: (provided) => ({
+              ...provided,
+              background: "white",
+            }),
+            multiValue: (provided) => ({
+              ...provided,
+              background: "primary.400",
+              color: "white",
+            }),
+            multiValueRemove: (provided) => ({
+              ...provided,
+              display: "none",
+              visibility: "hidden",
+            }),
+          }}
+          components={{
+            Menu: () => null,
+            DropdownIndicator: () => null,
```

Comment:
how come you remove these? is it to match the design? I found it harder to use the input without these components, but might just be me

Reply:
Yes, to match the designs -- it's not so much a dropdown select box as it is a box to hold multiple tags that the user can create
<img width="401" alt="Screen Shot 2022-06-19 at 2 24 25 AM" src="https://user-images.githubusercontent.com/99051851/174470403-be763911-05b7-40e7-ac15-1df8eb530f8c.png">
The 'x' I'm leaving it to be able to clear all tags at once if desired.